tests/test_jjb.py

```diff
@@ -44,6 +50,15 @@ def test_calamari_clients(self):
     def test_non_ascii(self):
         self.run_jjb('non-ascii', 'テスト')
 
-    def test_indentation(self):
-        self.compare_jjb_output('indentation', 'indentation')
+    def test_indentation_with_tabs(self):
+        yaml.add_representer(str, get_str_presenter(should_replace_tabs=True))
+        if PY2:
+            yaml.add_representer(unicode, get_str_presenter(should_replace_tabs=True))
+        self.compare_jjb_output('indentation_with_tab', 'indentation_with_tab')
 
+
+    def test_indentation_without_tabs(self):
+        yaml.add_representer(str, get_str_presenter(should_replace_tabs=False))
```

Comment:
now we have the same piece of code in 4 places, maybe it should be extracted to a function in cli.py?